stable/prometheus-adapter/values.yaml

```diff
@@ -40,7 +40,7 @@ resources:
 
 rules:
   default: true
-  custom: {}
+  custom: []
```

Comment:
Please update the readme to match too. TIA

Reply:
Thanks, done!